lib/users/routes.js

```diff
@@ -261,9 +312,46 @@ const save = (req, res, next) => {
   return res.redirect(referrer);
 };
 
+/**
+ * Save first user
+ * @param {object} req - HTTP Request
+ * @param {object} res - HTTP Response
+ * @param {object} next - Express callback
+ *
+ * @returns {promise} database promise
+ *
+ */
+const first = (req, res, next) => {
+  return database.select('*').from('users').then(rows => {
+    // if any users in DB, we give them a shameful error
+    if (rows.length > 1) {
```

Comment:
Shouldn't this never happen? I'd prefer to see some access control on the first time user creation routes that only allow access if the number of users in a database is greater than zero, as opposed to keeping that page accessible.